static/js/stat_var_hierarchy/stat_var_section_input.tsx

```diff
@@ -97,12 +109,51 @@ export class StatVarSectionInput extends React.Component<
         <label
           className={this.state.checked ? "selected-node-title" : ""}
           htmlFor={sectionId}
+          onMouseMove={
+            !this.props.statVar.hasData ? this.mouseMoveAction : null
+          }
+          onMouseOut={() => hideTooltip()}
         >
           {this.props.statVar.displayName}
         </label>
       </form>
     );
   }
+
+  private mouseMoveAction = (e) => {
+    let html = "loading stat var summary...";
+    if (!_.isEmpty(this.props.summary)) {
+      let availablePlaceTypes = Object.keys(
+        this.props.summary.placeTypeSummary
+      );
+      if (this.context.statVarHierarchyType === StatVarHierarchyType.MAP) {
+        availablePlaceTypes = availablePlaceTypes.filter(
+          (placeType) =>
+            placeType in USA_CHILD_PLACE_TYPES && placeType !== "Country"
+        );
+      }
+      html =
+        availablePlaceTypes.length > 0
+          ? "This stat var has no data for any of the chosen places." +
+            "You can try these types of places instead. <ul>"
+          : "Sorry, this stat var is not supported by this tool.";
+      for (const placeType of availablePlaceTypes) {
```

Comment:
another thing is to the order of place types, can defer to later PR

Reply:
Ok, what would be the order of place types?